Network/Wai/Middleware/Static.hs

```diff
@@ -162,35 +185,50 @@ static = staticPolicy mempty
 static' :: CacheContainer -> Middleware
```

Comment:
This function is superseded by `staticWithOptions`, correct? If so, I think we should deprecate this function in favor of `staticWithOptions` and remove it in the next major release. Can you add such a deprecation notice to `static'`s Haddocks?

Reply:
I added deprecation notices. I wanted to leave decisions about backwards incompatible changes to you.